devicetypes/smartthings/zigbee-battery-accessory-dimmer.src/zigbee-battery-accessory-dimmer.groovy

```diff
@@ -118,9 +126,50 @@ def handleSengledSwitchEvents(descMap) {
 	}
 }
 
+def handleCentraliteSmartSwitchEvents(descMap) {
+	if (descMap.clusterInt == 0x0006) {
+		if (descMap.commandInt == 0x01) {
+			sendEvent(name: "switch", value: "on")
+		} else if (descMap.commandInt == 0x00) {
+			sendEvent(name: "switch", value: "off")
+		}
+	} else if (descMap.clusterInt == 0x0008) {
+		def currentLevel = device.currentValue("level") as Integer ?: 0
+		if (descMap.commandInt == 0x05) {
```

Comment:
Is this how this is behaving?

0x05 is LEVEL_MOVE_ONOFF_COMMAND and 0x01 is LEVEL_MOVE_COMMAND. There should be a direction indication as the first byte in data (and rate as the second).

They should generally behave the same way, but with LEVEL_MOVE moving to a level, and LEVEL_MOVE_ONOFF sending on or off if the move would change the state of the device (i.e. if the device is off and you move to a level other than zero turn it on, and if the device is on and move to 0 then turn it off).

Reply:
If it is doing its own thing and behaving this way, it should definitely be documented in a comment.

Reply:
Ok, you're right! I've added a comment now because it is a specific behavior for this device.